Sources/Models/API/ProductAttributesReadAPI.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ProductAttributesAPIkeysJSON.swift
+//  OpenFoodFacts
+//
+//  Created by Alexander Scott Beaty on 9/29/20.
+//
+
+import Foundation
+
+// MARK: - ProductAttributes keys
+struct ProductAttributesJson {
+    static let BarcodeKey = "code"
+    static let AttributeGroupsKey = "attribute_groups"
+
+    static func attributeGroupsLang() -> String {
+        return AttributeGroupsKey + "_" + Bundle.main.currentLocalization
+    }
```

Comment:
- Created this method to append two-letter language code when requesting or mapping "attribute_groups."
- Using `currentLocalization` because it reflects the language set in the app's settings.